src/Pond.tsx

```diff
@@ -32,12 +32,38 @@ type PondProps = {
   loadComponent?: JSX.Element
   /** Error callback the the pond is not able to reach actyxOS locally */
   onError?: (error: unknown) => void | JSX.Element
-  /** optional url to overwrite to local connection and connect to an other peer (default: `ws://localhost:4243/store_api`) */
-  url?: string
-  /** Hook, when the connection to the store is closed */
-  onStoreConnectionClosed?: () => void
-  /** Advanced configuration options for the Pond. */
-  pondOptions?: PondOptions
+  /**
+   * Optionally change the Actyx endpoint the Pond connects to
+   * and provide an error handler if the connection to Actyx is dropped.
+   * Defaults to `{}`
+   * */
+  connectionOpts?: ActyxOpts
+  /** 
+   * Advanced Pond configuration options.
+   * Defaults to `{}`
+   *  */
+  opts?: PondOptions
+  /**
+   * Manifest describing an Actyx application. Used for authorizing API access.
+   * Use `com.example.<somestring>` as `appId` for testing and development purposes,
+   * so you don't require a signed certificate.
+   * 
+   * Defaults to
+   * ```
+   * {
+   *   appId: 'com.example.react-pond-example',
+   *   displayName: 'React Pond Example',
+   *   version: '0.0.1'
+   * }
+   * ```
+   * */
+  manifest?: AppManifest
+}
+
+const defaultManifest : AppManifest = {
```

Comment:
this is somehow magic. I would prefer a required manifest.

Reply:
Ok, you saw this ;-) ☝️